### PR TITLE
Minor documentation corrections

### PR DESCRIPTION
**Pull request by megazord**
1. Correct link to Activator download
2. Better organization and description of tutorials list
3. Change order of pages for Hacking section
4. Small format corrections

This corrects some of the commends mades at PR #4231.

### CODE REVIEW

File: documentation/manual/gettingStarted/Tutorials.md [hunk 1]
```diff
@@ -11,9 +11,15 @@ The template directory allows you to browse templates, preview the tutorials tha
...
 
-* [Reactive Maps (Scala)](https://typesafe.com/activator/template/reactive-maps)
-* [Reactive Maps (Java 8)](https://typesafe.com/activator/template/reactive-maps-java)
-* [Super Slick Play (Scala)](https://typesafe.com/activator/template/play-slick)
-* [Play with Spring Data JPA (Java)](https://typesafe.com/activator/template/play-spring-data-jpa)
-* [AngularJS Play Mongo (Scala)](https://typesafe.com/activator/template/modern-web-template)
-* [Play Mongo Knockout (Scala)](https://typesafe.com/activator/template/play-mongo-knockout)
+## Scala
+
+1. [Play Scala Intro](https://typesafe.com/activator/template/play-scala-intro): Introduction to create a simple CRUD application with Play Scala.
```
Comment (cchantep) [inline documentation/manual/gettingStarted/Tutorials.md]: Does it need to be ordered?

**Comment by megazord** [comment]
I've ordered them to express the increasing complexity of the tutorials, but it is OK to me if this list unordered.

File: documentation/manual/gettingStarted/Tutorials.md (hunk 1 above)
Comment (megazord) [inline documentation/manual/gettingStarted/Tutorials.md]: I've ordered them to express the increasing complexity of the tutorials, but it is OK to me if this list unordered.

Comment (cchantep) [inline documentation/manual/gettingStarted/Tutorials.md]: For me, the order is not stating clearly that it is about complexity.

**Comment by megazord** [comment]
@cchantep the tutorials items are not ordered anymore.

**Comment by cchantep** [comment]
@play-pull-request-validator

**Pull request merged**